apps/storefront/components/productList/FilteredProductList/FilteredProductList.tsx

```diff
@@ -178,6 +165,19 @@ export function FilteredProductList({
             onRemoveAttribute={removeAttributeFilter}
           />
         )}
+
+        <hr style={{ marginTop: "30px", color: "#F0F0F0" }} />
+        <p
+          style={{
+            color: "#072137",
+            fontSize: "16px",
+            fontWeight: 600,
+            border: "none",
+            margin: "50px 0 32px",
+          }}
```

Comment:
I don't think that it is a good idea to use styles like that.
The project styles become inconsistent
If we need responsive design in the future we can't do it like that or it will be very hard  
My suggestion is to continue using the old-style system of the project